ui/.gitignore

```diff
@@ -0,0 +1,14 @@
+# dependencies
+/node_modules
+
+# testing
+/coverage
+
+# build
+/build
+/dist
+
+# local logs
+npm-debug.log*
```

Comment:
Just curious, why is it `.log*`? Can something follow `.log`?

Reply:
I guess so. These filenames were in a few different recommended gitignore files I found, and they consistently included the asterisk at the end.